tests/benchmarks/unpack_sequence.py

```diff
@@ -148,10 +148,10 @@ def do_unpacking(iterations, to_unpack):
         a, b, c, d, e, f, g, h, i, j = to_unpack
         a, b, c, d, e, f, g, h, i, j = to_unpack
         a, b, c, d, e, f, g, h, i, j = to_unpack
+        """
```

Comment:
My interpretation of this is that as a result of this PR, unpacking takes *more* code (slightly more, anyway). Where does the extra overhead come from? Is it avoidable at all?

Reply:
I'd tried several variations, but could not get the output significantly smaller than this. Improving the validation of how many values to unpack is a significant part of the size increase.

Note: while I don't have all my calculations handy... each line in this test is making 10 assignments, so 1370 assignments are being made before hitting the GOTO limit of +/-32k. That's less than 24 bytes being 'spent' per assignment on average. Even if that limit was dealt with, there would still be the 64k function-size limit.

Reply:
I added a mention of the need for this in the relevant commit message.